worlds/checksmate/test/__init__.py

```diff
@@ -0,0 +1,16 @@
+from typing import ClassVar
+
+from test.bases import WorldTestBase
+from .. import CMWorld
+
+
+class CMTestBase(WorldTestBase):
+    game = "ChecksMate"
+    world: CMWorld
+    player: ClassVar[int] = 1
+
+    def world_setup(self, *args, **kwargs):
+        self.options["accessibility"] = "minimal"
```

Comment:
This modifies the "global" `WorldTestBase.options`. You'd need to copy the dict before modifying, however I think that forcing accessibility=minimal isn't great either way.

Reply:
Both aspects of this comment have been resolved.